openwisp_users/apps.py

```diff
@@ -130,6 +137,18 @@ def connect_receivers(self):
             dispatch_uid='make_first_org_user_org_owner',
         )
 
+    @classmethod
+    def handle_organization_update(cls, instance, **kwargs):
+        Organization = instance._meta.model
```

Comment:
This will avoid the extra query for new organizations.

```suggestion
        # If it's a new organization, we don't need to update any cache
        if instance._state.adding:
            return
        Organization = instance._meta.model
```

Reply:
Yes adding this reduced queries in some testcases I will update that too.